src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,90 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+public class StringTest {
+    @DisplayName("split을 통해 콤마로 구분된 문자열을 각각의 원소로 나눌 수 있다.")
+    @Test
+    void splitTwoValues() {
+        //GIVEN
+        String src = "1,2";
+
+
+        //WHEN
+        String[] result = src.split(",");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?

Reply:
given/when/then 의 구조를 의식하다보니 형태에 맞추려고 한 흔적이 보이네요.
훨씬 더 간결하게 표현할 수 있을 것 같습니다.